help/elements.json

```diff
@@ -1,18 +1,18 @@
 {
     "*.Condition": {
         "description": "Optional expression evaluated to determine whether the target element should be evaluated",
-        "help": "https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-conditions"
+        "helpLink": "https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-conditions"
```

Comment:
Make this name consistent with other files. It is called `helpLink` everywhere else